src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,70 @@
+package mate.academy;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import mate.academy.exceptions.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_INIT_PATH = "src/main/resources/init_db.sql";
+    private static final String TABLE_BOOKS_NAME = "books";
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "artem");
+        DB_PROPERTIES.put("password", "Pssmgk_82");
+
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+            initializeTable();
```

Comment:
```suggestion
```
remove this logic

Reply:
++, removed this expression (i didn't get if i had to remove the logic related to create the table and invoke this function in Main class)